rmarkdown/rmarkdown-accessible.qmd

```diff
@@ -0,0 +1,538 @@
+|                                                 | $$                                                                       |
++-------------------------------------------------+--------------------------------------------------------------------------+
+|     horizontal rule:                            | horizontal rule:                                                         |
+|     ---                                         |                                                                          |
+|                                                 | ------------------------------------------------------------------------ |
++-------------------------------------------------+--------------------------------------------------------------------------+
+|     |Right|Left|Default|Center|                 | | Right | Left | Default | Center |                                      |
+|     |----:|:---|-------|:----:|                 | |------:|:-----|---------|:------:|                                      |
+|     |12   |12  |12     |12    |                 | |    12 | 12   | 12      |   12   |                                      |
+|     |123  |123 |123    |123   |                 | |   123 | 123  | 123     |  123   |                                      |
+|     |1    |1   |1      |1     |                 | |     1 | 1    | 1       |   1    |                                      |
+|                                                 |                                                                          |
+|     Table: Caption text, example rendered table | : Caption text, example rendered table.                                  |
++-------------------------------------------------+--------------------------------------------------------------------------+
+| HTML Tabsets                                    | ::: panel-tabset                                                         |
+|                                                 | ## Plots                                                                 |
+|     # Results                                   |                                                                          |
+|     ## Plots text                               | text                                                                     |
+|     text                                        |                                                                          |
+|                                                 | ## Tables                                                                |
+|     ## Tables                                   |                                                                          |
+|     more text                                   | more text                                                                |
+|                                                 | :::                                                                      |
```

Comment:
Markdown doesn't match the output here, e.g. missing `::: panel-tabset` in left hand column